ChallengesWithTestsMarkVII.Tests/ChallengesWithTestsMarkVII.Tests.sln

```diff
@@ -1,17 +1,28 @@
 
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
+<<<<<<< HEAD
```

Comment:
This is partly my fault for not helping catch this when we were resolving the merge conflict. These merge conflicts should  have been cleaned up.